src/LoginFormContainer.test.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import { fireEvent, render } from '@testing-library/react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { changeLoginFields, setAccessToken } from './actions';
+
+import LoginFormContainer from './LoginFormContainer';
+
+jest.mock('react-redux');
+
+describe('LoginFormContainer', () => {
+  const dispatch = jest.fn();
+
+  beforeEach(() => {
+    dispatch.mockClear();
+
+    useDispatch.mockImplementation(() => dispatch);
+
+    useSelector.mockImplementation((selector) => selector({
+      loginFields: {
+        email: '',
+        password: '',
+      },
+      accessToken: given.accessToken,
+    }));
+  });
+
+  context('with accessToken', () => {
+    given('accessToken', () => ({
+      accessToken: 'qwer',
+    }));
```

Comment:
```suggestion
    given('accessToken', () => 'ACCESS_TOKEN');
```

요렇게 하면 더 간단하고 의미가 드러나죠ㅎㅎ